common/js/injected_script.js

```diff
@@ -248,6 +248,8 @@ class InjectedScript {
       );
       result = result.concat(shadowRootResults);
     }
+
+    if (root.children === undefined) return result;
```

Comment:
Should we use [`hasChildNodes`](https://developer.mozilla.org/en-US/docs/Web/API/Node/hasChildNodes)?

```suggestion
    if (!root.hasChildNodes()) return result;
```

Reply:
We can do that. See [a03cc81](https://github.com/grafana/xk6-browser/pull/1049/commits/a03cc813f08b3de530192c9e94a9bf8c2166382d).